back/engines/commercial/multi_tenancy/app/models/multi_tenancy/extensions/app_configuration.rb

```diff
@@ -70,7 +70,7 @@ def from_tenants(tenants)
             .join(' UNION ')
 
           app_configuration = ActiveRecord::Base.connection.execute(sql_query)
-          app_configuration.map { |record| new(record) }
+          app_configuration.map { |record| instantiate(record) }
```

Comment:
The removed gem used to JSON parse any attributes it was validating and reassign the result to the object. The new implementation no longer does this, as it doesn't seem to be needed for validation.

Turns out that this was masking a bug, where we weren't instantiating these AppConfiguration records properly. Using `new`, not all AR logic was executing. The JSONB columns came back as String instead of Hash.

This fixes it.

Reply:
Nice catch :)